backend/routes/db.py

```diff
@@ -19,11 +19,12 @@
 
 @router.get('/last-refreshed')
 def last_refreshed_DB():
+    """Check when database was last refreshed."""
     with get_db_connection() as con:
         q = """
               SELECT value
               FROM public.manage
-              WHERE key = 'last_refreshed_DB'
+              WHERE key = 'last_refresh_success'
```

Comment:
@hugwuoke `last_refreshed_DB ` 2/2: This wasn't something I thought of during our meeting, nor is it something that our unit test(s) might have caught. I'm going to be deleting this `last_refreshed_DB` soon now that we no longer need it.